app/models/notification_blueprint.rb

```diff
@@ -10,7 +10,6 @@ class NotificationBlueprint < ActiveRecord::Base
   has_many :notifications, :dependent => :destroy
   store :actions, :accessors => [:links], :coder => JSON
 
-  validates :message, :presence => true
```

Comment:
since uinotifiations can override the blueprint when creating the message, I'm not in favor of removing the validation. lets create some dummy sample instead of reducing the validation for the "edge" case.